edb/schema/functions.py

```diff
@@ -717,6 +717,14 @@ class CallableObject(
     abstract = so.SchemaField(
         bool, default=False, inheritable=False, compcoef=0.909)
 
+    fallback = so.SchemaField(
```

Comment:
Please add a comment describing this field.  Also, don't forget to bump CATALOG_VERSION since this adds a new property to the reflection schema.

Reply:
Comment added and catalog version bumped.